manifests/charts/gateways/istio-egress/templates/autoscale.yaml

```diff
@@ -1,6 +1,6 @@
 {{ $gateway := index .Values "gateways" "istio-egressgateway" }}
 {{- if and $gateway.autoscaleEnabled $gateway.autoscaleMin $gateway.autoscaleMax }}
-apiVersion: autoscaling/v2beta1
+apiVersion: autoscaling/v2beta2
```

Comment:
v2beta2 just pushes us back 1 release to 1.26. So we may need to have support for v2beta2 and v2 unfortunately... Or just v2beta1 and v2 possibly. See https://kubernetes.io/docs/reference/using-api/deprecation-guide/#horizontalpodautoscaler-v126

Reply:
Which can just be a simple `if` in the api version I think - the schema is probably the same

Reply:
yep I think the schema is the same between v2beta2 and v2